tensorboard/plugins/debugger/README.md

```diff
@@ -166,10 +166,21 @@ For example, suppose a tensor has a shape of (500, 100), applying a slicing of `
 
 For each tensor, the time axis (history of the tensor's execution) is treated as an 1D array. Numpy-style slicing can be applied to time. For example, the default slicing of `-1` selects the most recent value. However, if the user changes that slicing parameter to `:`, the full history of the tensor will be shown (and the rank of the tensor being visualized is increased by 1).
 
-# Limitations
+# Frequently Asked Questions (FAQ)
+
+## Q: How to exit the debugging?
+
+Answer: Follow these steps to interrupt your TensorFlow program being debugged
+and the TensorBoard process running the Debugger Plugin (*in that order*):
+
+1. Send `SIGINT` to the TensorFlow program being debugged, e.g., by using
+   `Ctrl+C`.
+2. Send `SIGINT` to the TensorBoard process running the Debugger Plugin, e.g.,
+   by using `Ctrl+C`.
+
+# Limitations and Unknown Issues
+
+The Debugger Plugin has the following limitations and known issues. We plan to
+fix them in future releases.
 
-* Hitting Ctrl+C (issuing a SIGINT signal) might fail to terminate execution for a model that is instrumented with
-  `TensorBoardDebugWrapperSession` or its corresponding hook. The same limitation may be present in the tensorboard
-  process as well. In those cases, the user must manually
-  [kill](https://www.linux.com/learn/intro-to-linux/2017/5/how-kill-process-command-line) the processes.
 * The debugger dashboard does not yet support multiple users debugging at once.
```

Comment:
Maybe change this phrasing to "debugger plugin" to match other mentions.

Reply:
Done.